blueflood-core/src/main/java/com/rackspacecloud/blueflood/cache/SafetyTtlProvider.java

```diff
@@ -27,79 +29,51 @@
 import com.rackspacecloud.blueflood.utils.TimeValue;
 import java.util.concurrent.TimeUnit;
 
+/**
+ * Implementation that provides the safe lower bound for the metric TTLs.
+ *
+ * Even though the methods of this class return {@link Optional}, they always return a present value.
+ */
 public class SafetyTtlProvider implements TenantTtlProvider {
+    private static final TimeValue DAY = new TimeValue(1, TimeUnit.DAYS);
     private final ImmutableTable<Granularity, RollupType, TimeValue> SAFETY_TTLS;
-    private final TimeValue STRING_TTLS = Constants.STRING_SAFETY_TTL;
-    private final TimeValue CONFIG_TTL = new TimeValue(Configuration.getInstance().getIntegerProperty(TtlConfig.TTL_CONFIG_CONST), TimeUnit.DAYS);
-    private final boolean ARE_TTLS_FORCED = Configuration.getInstance().getBooleanProperty(TtlConfig.ARE_TTLS_FORCED);
+    private final TimeValue STRING_TTL = Constants.STRING_SAFETY_TTL;
 
     private static final SafetyTtlProvider INSTANCE = new SafetyTtlProvider();
 
-    public static SafetyTtlProvider getInstance() {
+    static SafetyTtlProvider getInstance() {
         return INSTANCE;
     }
 
-    private SafetyTtlProvider() {
+    SafetyTtlProvider() {
         ImmutableTable.Builder<Granularity, RollupType, TimeValue> ttlMapBuilder =
                 new ImmutableTable.Builder<Granularity, RollupType, TimeValue>();
 
         for (Granularity granularity : Granularity.granularities()) {
             for (RollupType type : RollupType.values()) {
-                try {
-                    MetricColumnFamily metricCF = CassandraModel.getColumnFamily(RollupType.classOf(type, granularity), granularity);
-                    TimeValue ttl = new TimeValue(metricCF.getDefaultTTL().getValue() * 5, metricCF.getDefaultTTL().getUnit());
-                    ttlMapBuilder.put(granularity, type, ttl);
-                } catch (IllegalArgumentException ex) {
-
+                if (type == RollupType.NOT_A_ROLLUP) {
+                    continue;
                 }
+                MetricColumnFamily metricCF = CassandraModel.getColumnFamily(RollupType.classOf(type, granularity), granularity);
```

Comment:
@DanielNi  - I missed this before, but FYI, we are currently moving to the Datastax driver, which will allow us to upgrade cassandra.  MetricColumnFamily sub-classes an Astyanax-specific class.  Long-term we'll want to wrap driver-specific types in a class, e.g., https://github.com/rackerlabs/blueflood/blob/master/blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DRawIO.java

However, this will probably happen after we accept this PR, so we can handle this later.

cc - @ChandraAddala @shintasmith @VinnyQ